src/framework/components/anim/component.js

```diff
@@ -636,9 +646,13 @@ class AnimComponent extends Component {
      * @name AnimComponent#setTrigger
      * @description Sets the value of a trigger parameter that was defined in the animation components state graph to true.
      * @param {string} name - The name of the parameter to set.
+     * @param {boolean} singleFrame - If true, this trigger will be set back to false at the end of the animation update. Defaults to false.

```

Comment:
JSdocs should show this is optional?
```suggestion
     * @param {boolean} [singleFrame] - If true, this trigger will be set back to false at the end of the animation update. Defaults to false.

```

Reply:
Good catch, cheers